samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -112,6 +118,13 @@ func WithSamplingRefreshInterval(samplingRefreshInterval time.Duration) Option {
 	})
 }
 
+// WithSamplingLogger creates a Option that initializes sampling logger.
+func WithSamplingLogger(logger Logger) Option {
```

Comment:
This should accept a `"github.com/go-logr/logr".Logger` (similar to [this](https://github.com/open-telemetry/opentelemetry-go/blob/096a1624184285edbf9c82c53ea0b068895915ef/internal_logging.go#L23-L26)).

There should not be a need to export our own `Logger` interface here.

Reply:
done.